kernel/Kconfig

```diff
@@ -928,4 +928,16 @@ config THREAD_LOCAL_STORAGE
 
 endmenu
 
+menu "Device Options"
+
+config HAS_DYNAMIC_DEVICES
```

Comment:
as I mentioned, I think the "dynamic devices" term is confusing. Maybe it's because it's not explained anywhere what a "dynamic device" is.

Reply:
that is fair, what a about `DYNAMIC_DEVICE_DEPENDENCIES`  ?

Reply:
changed to `HAS_DYNAMIC_DEVICE_HANDLES`